eng/pipelines/pull-request-common.yml

```diff
@@ -58,10 +58,9 @@ steps:
 
   - script: node common/scripts/install-run-rush.js e2e
     displayName: E2E Tests
-    # Skipping the e2e test on the prepare publish branch as there is a bug in the npm version installed on node 16
+    # Skipping the e2e test on node 16.x as there is a bug in the npm version installed on node 16
     # causing npx to try to download the unpublished version of the cli.
-    condition: eq('${{ parameters.nodeVersion }}', '18.x')
-    # condition: and(not(startsWith(variables['System.PullRequest.SourceBranch'], 'publish/')), not(eq('${{ parameters.nodeVersion }}', '16.x')))
+    condition: ne(variables.nodeVersion, '16.x')
```

Comment:
Do you know the specific bug in the npm version installed with Node 16?

Reply:
I think it is the npm version but on windows `npm install -g npm` doesn't really update the npm version on Azure Devops there is a whole thing about it so it would still fail. Maybe this has been fixed since then though, let me try first

Reply:
hhm seems like it is working now so maybe i can just remove this stupid condition.